meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   build:
     - python
     - yaml
-    - openssl 1.0.1g
+    - openssl
```

Comment:
Good call.  There is a minimum requirement though, so should probably be `openssl >=1.0.1g` so we ensure at least that version is available.